### DISCUSSION TITLE
G6如何渲染带有latex的文本?

### DISCUSSION THREAD

**Discussion by CyNotes** [antvis/G6#4994] [Q&A]
我现在希望引入mathjax来渲染数学公式latex, 但G6的自定义节点似乎没法支持mathjax产生的jsx, 怎样才能渲染latex呢?  [沙盒链接](https://codesandbox.io/s/eloquent-bird-wlhy75?file=/src/App.js)

**Comment by Aarebecca** [comment]
你好，G6 支持 JSX 节点的特性会在后续推出，到时可在自定义节点中渲染相关内容 https://github.com/antvis/G6/pull/5079

**Comment by Aarebecca** [comment]
可以参考这个案例使用自定义节点来渲染 latex 语法 https://g6-next.antv.antgroup.com/examples/item/customNode/#react-node
